esp/applications/eclwatch/eclwatch.yaml

```diff
@@ -2,6 +2,9 @@ eclwatch:
    WsSMC:
       ActivityInfoCacheSeconds: 10
       ActivityInfoCacheAutoRebuildSeconds: 120
+   WsCloud:
+      POSInfoCacheSeconds: 10
```

Comment:
Did you mean to spell PODInfoCacheSeconds (D or S)?

Reply:
Will fix.